docs/src/docs/examples.clj

```diff
@@ -12,11 +12,12 @@
 
 ;; tag::start-standalone-node[]
 (defn start-standalone-node ^crux.api.ICruxAPI [storage-dir]
-  (crux/start-node {:crux.node/topology '[crux.standalone/topology]
+  (crux/start-node {:crux.node/topology 'crux.standalone/topology
                     :crux.node/kv-store 'crux.kv.memdb/kv
                     :crux.kv/db-dir (str (io/file storage-dir "db"))
                     :crux.standalone/event-log-kv-store 'crux.kv.memdb/kv
                     :crux.standalone/event-log-dir (str (io/file storage-dir "event-log"))}))
+(def node (start-standalone-node "crux-store"))
```

Comment:
so you can just copy and paste

Reply:
I'd come down on the side of not including these top-level side-effects in our examples, particularly given this opens a closable resource - it's ok for REPL usage but outside of this we should be encouraging users to use `with-open`, or a component-like framework that ensures resources are closed properly.

(also, for internal dev experience, the tests evaluate this namespace to ensure the examples still work - opening a node here would likely affect being able to run the tests multiple times)

Reply:
@jarohen good, then two options are left – discard the line or wrap into a `comment` form

Reply:
@jarohen I've left a commented out suggestion, but feel free to drop it